README.md

```diff
@@ -546,7 +546,7 @@ strategies:
 When the descheduler decides to evict pods from a node, it employs the following general mechanism:
 
 * [Critical pods](https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/) (with priorityClassName set to system-cluster-critical or system-node-critical) are never evicted.
-* Pods (static or mirrored pods or stand alone pods) not part of an RC, RS, Deployment or Job are
+* Pods (static or mirrored pods or stand alone pods) not part of an ReplicationController, ReplicaSet, StatefulSet, Deployment or Job are
```

Comment:
We might also write a simple unit test for the stategy evicting SS pods so the claim is not "just" on the paper.

Reply:
@ingvagabund do you want the unit/e2e tests in this PR or a separate PR?

Reply:
In this PR preferably

Reply:
Somewhere into https://github.com/kubernetes-sigs/descheduler/blob/master/pkg/descheduler/evictions/evictions_test.go#L72 will do with new `GetStatefulSetOwnerRefList()` into https://github.com/kubernetes-sigs/descheduler/blob/master/test/test_utils.go#L83. e2e would be better though given this is just to suppose the claims in the document, unit test is sufficient.

Reply:
I'll write the unit tests for StatefulSets as par of this PR. I'll create a separate issues for tracking the creation of StatefulSet e2e tests.

Reply:
Fixed in commit 85fcd156277382a8064af4e565d686cdf2594ca2